kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -37,4 +53,49 @@ server {
         proxy_set_header Host              $http_host;
         proxy_pass                         $nginx_storage_backend;
     }
+
+    location @error_400 {
+        default_type application/json;
+        return 400 '{"error_message": "Bad request", "error_code": "bad-request"}';
+    }
+
+    location @error_401 {
+        default_type application/json;
+        return 401 '{"error_message": "Access denied.", "error_code": "unauthorized"}';
+    }
+
+    location @error_403 {
+        default_type application/json;
+        return 403 '{"error_message": "Not permission for this page", "error_code": "forbidden"}';
+    }
+
+    location @error_404 {
+        default_type application/json;
+        return 404 '{"error_message": "The page does not exist", "error_code": "not-found"}';
+    }
+
+    location @error_408 {
+        default_type application/json;
+        return 408 '{"error_message": "The server timed out waiting for the request", "error_code": "request-timeout"}';
+    }
+
+    location @error_500 {
+        default_type application/json;
+        return 500 '{"error_message": "The server occur unexpected error", "error_code": "internal-server-error"}';
```

Comment:
Please use the standard messages. This should be for example `Internal server error`.